libgdf/README.md

```diff
@@ -56,25 +56,25 @@ the dependencies.
 
 ```bash
 # create the conda environment (assuming in build directory)
-$ conda env create --name libgdf_dev --file conda_environments/dev_py35.yml
+$ conda env create --name cudf_dev --file ../conda_environments/dev_py35.yml
 # activate the environment
-$ source activate libgdf_dev
+$ source activate cudf_dev
 # when not using default arrow version 0.10.0, run
 $ conda install pyarrow=$ARROW_VERSION -c conda-forge
 ```
 
-This installs the required `cmake` and `pyarrow` into the `libgdf_dev` conda
+This installs the required `cmake` and `pyarrow` into the `cudf_dev` conda
```

Comment:
PyArrow isn't needed for libgdf as far as I'm aware, but boost is which is pulled via the conda file.

Reply:
All I did was change the name of the env here in anticipation of the cudf C++ refactor. I don't use that conda install command at all myself. Not sure why it's in the readme. Removing that could be another PR by someone who knows. :)